### PR TITLE
Switch to using opencontainers/selinux

### PR DESCRIPTION
**Pull request by rhatdan**
We have moved selinux support out of opencontainers/runc into its
own package.  This patch moves to using the new selinux go bindings.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: cmd/ocid/main.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/containers/storage/pkg/reexec"
 	"github.com/kubernetes-incubator/cri-o/server"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```
Comment (runcom) [inline cmd/ocid/main.go]: this alias shouldn't be necessary as `go-selinux` just exports the `selinux` package directly

File: server/config.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 	"io/ioutil"
 
 	"github.com/BurntSushi/toml"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```
Comment (runcom) [inline server/config.go]: same here afaict

**Comment by runcom** [comment]
@rhatdan seems like the actual vendoring code is missing in this PR

**Comment by mrunalp** [comment]
@rhatdan Can you rebase this to master which now has a fix for the timeout issue?

**Comment by rhatdan** [comment]
Done, we will see if the tests pass.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**